modules/doc/content/getting_started/installation/inl_hpc_install_moose.md

```diff
@@ -4,32 +4,66 @@ The following instructions are for those operating on [!ac](INL) [!ac](HPC) mach
+!style halign=left
 The development environment is deployed using containerization, which is a deployment
-practice that bundles all application (i.e., MOOSE or a MOOSE-based app) dependencies need to run
-on any host. This means that applications built within this environment can be executed
-across all INL HPC clusters. For example, you can build an application within a compute node on
-Sawtooth and execute the same executable of the application (providing you follow the instructions
-that follow) on both Bitterroot and Lemhi.
+practice that bundles all application and application dependencies (i.e., MOOSE or a MOOSE-based
+app), needed to run on any host. This means that applications built within this environment can be
+executed across all [!ac](INL) [!ac](HPC) clusters. For example, you can build an application within
+a compute node on Sawtooth and execute the same executable of the application (providing you follow
+the instructions that follow) on both Bitterroot and Lemhi.
+
+## Obtain MOOSE
+
+!style halign=left
+If you have not yet cloned MOOSE or an application that uses moose, do this now according to that
+application's instructions. Otherwise, skip to
+[Loading the Environment](inl_hpc_install_moose.md#envloading) below.
 
-## Versioning id=versioning
+!alert! tip title=Use `/scratch` while using INL HPC resources
+Use the available `/scratch/<your INL HPC User ID>/` location whenever possible. The scratch volume
+offers higher performance than what exists for your `/home/` directory.
+!alert-end!
 
-The version of the dependencies used within MOOSE (PETSc, libMesh, etc) are tied directly
-to a given version of MOOSE. Given a version of MOOSE (or a MOOSE-based application), we produce
-a version of the development environment that contains the correct version of these dependencies.
-Whenever you are compiling MOOSE or a MOOSE-based application, you should use the corresponding
-version of the development environment.
+!template load file=installation/clone_moose.md.template PATH=/scratch/<your INL HPC User ID>/projects
 
-In order to determine the version of the development environment to use when building, run the
-following within the MOOSE directory:
+## Loading the Environment id=envloading
+
+!style halign=left
+Figuring out the right environment to load is a two-step process involving our Versioner script,
+and `module load` commands:
+
+### Versioner id=versioning
+
+!style halign=left
+To aid you in selecting the right container to use at any given commit, we created a script called
+`versioner.py`. Versioner returns a hash which we use as a naming suffix for the module you will be
+loading. Your copy of MOOSE, and all the accompanying submodules (recursively), contribute to the
+generation of this hash.
+
+Enter the `moose` directory where ever that may be, and run the following command (the following
+assumes you cloned MOOSE as directed in the previous step):
 
 ```bash
+cd /scratch/<your INL HPC User ID>/projects/moose
```

Comment:
People still can't do this on most hosts due to missing packages; we should probably add in a component about how to get a working env first. Because this won't work on its own